unime/src-tauri/tauri.conf.json

```diff
@@ -27,7 +27,7 @@
   "build": {
     "beforeDevCommand": "npm run dev",
     "beforeBuildCommand": "npm run build",
-    "devUrl": "http://localhost:5173",
+    "devUrl": "http://localhost:4173",
```

Comment:
Why change the dev server port?

Reply:
This was due to a port conflict while developing the ecommerce demo. The ecommerce site was already running on port 5173.